os/tools/check_output_size.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+###########################################################################
+#
+# Copyright 2021 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+
```

Comment:
Could you add description of this file? What does this file and what's the purpose?

Reply:
Done.

Reply:
@kishore-sn @thapav @amogh-samsung Could you review the description below?
```
# This script is for checking kernel partition is enough to accommodate kernel binary.
# For this, get the kernel partition size and kernel binary size, 
# and check the kernel binary size is smaller than kernel partition size.
```

Reply:
Changed. Thanks.

Reply:
The description looks ok. You could change the first line as.. 
`kernel partition is **large** enough to accommodate kernel binary.`

Reply:
Modified. Thanks.